template.yaml

```diff
@@ -37,16 +45,24 @@ Resources:
       Runtime: python3.9
       Environment:
         Variables:
+          apiChartOfAccounts: '/accounts'
+          apiValidTags: '/tags'
```

Comment:
I chose to hard-code all of the occurrences in the file because some are required to be hard-coded. also this is where we set the api route environment variables you asked about

Reply:
ok.